### PR TITLE
Closes #249 correction for rounding blog

### LINKED ISSUE #249: Documentation:  Rounding blog minor update needed
### Let us know where something needs a refresh or put your idea here!

A colleague of mine spotted that https://pharmaverse.github.io/blog/posts/2023-07-24_rounding/rounding.html is not quite accurate for SAS. It sas "SAS rounds half up" but actually SAS rounds away from zero.  You can see the difference in the negative numbers.  In half up, -1.15 would round up to -1.1 instead of -1.2.

CAMIS says the same: https://psiaims.github.io/CAMIS/Comp/r-sas_rounding.html

### PR DESCRIPTION
**Pull request by manciniedoardo**
Thank you for your Pull Request! We have developed this task checklist to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your blog post.

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `main` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update), and make sure the corresponding issue is linked in the Development section on the right hand side
- [x] Ensure your new post folder is of the form `"posts/zzz_DO_NOT_EDIT_<your post title>"`. This is so that the post date can be auto-updated upon the merge into `main`.
- [x] Run the script from `CICD.R` line by line to first check the spelling in your post and then to make sure your code is compatible with our code-style. Address any incongruences by following the instructions in the file! 
- [x] Choose (possibly several) `tag(s)` or categories from the current list: ` c("Metadata", "SDTM", "ADaM", "TLG", "Shiny", "Community", "Conferences", "Submissions", "Technical")` for your blog post. If you cannot find anything that fits your blog post, propose a new tag to the maintainers! Note: if you use a tag not from this list, the "Check Post Tags" CICD pipeline will error. We occasionally tidy up all `tags` for consistency.
- [x] Add a short description for your blog post in the `description` field at the top of the markdown document.
- [x] Blog post is *short*, *personalized*, *reproducible* and *readable*
- [x] Add a disclaimer at the top of your post, if appropriate (example: **Disclaimer** 
This blog contains opinions that are of the authors alone and do not necessarily reflect the strategy of their respective organizations.)
- [x] Address all merge conflicts and resolve appropriately
- [x] Assign two of us (@bms63, @manciniedoardo, @StefanThoma, @kaz462) as reviewers in the PR.
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

File: posts/2023-07-24_rounding/rounding.qmd
Comment (rossfarrugia) [inline posts/2023-07-24_rounding/rounding.qmd:28]:
```suggestion
-   SAS `round()` [rounds away from zero](https://documentation.sas.com/doc/en/pgmsascdc/9.4_3.5/lefunctionsref/p0tj6cmga7p8qln1ejh6ebevm0c9.htm#p0zr4tmk27unjwn1izg4801h5l46). So, `0.5` would be rounded to 1 whereas `-1.5` to `-2`.
```
The 2 examples given here currently would give the same result in SAS and R so maybe better to use one that more highlights the difference

Comment (rossfarrugia) [reply posts/2023-07-24_rounding/rounding.qmd:28]: this would also then be consistent with the examples in the below bullet

**Review by rossfarrugia — APPROVED** [review]
@manciniedoardo blog updates look fine to me but double check the CI spelling and links fails

**Comment by manciniedoardo** [comment]
@StefanThoma can we ignore the spelling?

**Comment by bms63** [comment]
I fixed the spelling in the #257 PR

**Comment by bms63** [comment]
@manciniedoardo I don't think we should ignore here, but I wish there was a way to ignore them in the spell check.  If we don't add then the spell checker will continuously fail

**Comment by manciniedoardo** [comment]
> @manciniedoardo I don't think we should ignore here, but I wish there was a way to ignore them in the spell check. If we don't add then the spell checker will continuously fail

ok, fixed, then i will merge this in and once we merge #257 the links should pass too.

**Pull request merged**
